pkg/registry/experimental/controller/etcd/etcd.go

```diff
@@ -88,6 +88,7 @@ func (r *ScaleREST) Update(ctx api.Context, obj runtime.Object) (runtime.Object,
 		return nil, false, errors.NewNotFound(extensions.Resource("replicationcontrollers/scale"), scale.Name)
 	}
 	rc.Spec.Replicas = scale.Spec.Replicas
+	rc.ResourceVersion = scale.ResourceVersion
```

Comment:
@bgrant0607 are we going to require every client submitting a scale request do a get first and handle retry on conflict, or allow an empty resource version if the client doesn't care about current state and just wants to submit a scale request? I'd expect to only set this if `len(scale.ResourceVersion) > 0`. Same goes for the Update methods on Deployments and ReplicaSet, I think